benchmarks/src/test/java/zipkin2/server/ServerIntegratedBenchmark.java

```diff
@@ -91,7 +91,7 @@ class ServerIntegratedBenchmark {
 
```

Comment:
Note: this bench doesn't complete yet, but I can work on that in another PR. Good news is it at least runs on Apple Silicon now!